packages/server-wallet/src/db/migrations/20201211180815_finalization_status.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from 'knex';
+
+const tableName = 'finalization_status';
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable(tableName, function(table) {
+    table.string('channel_id').notNullable();
+    table.integer('finalizes_at').notNullable();
+    table.integer('block_number').notNullable();
```

Comment:
These properties seem to be common among all channels. What's the argument for storing them in a separate table?

Skimming the rest of this PR, the logic would be simpler if these data were stored on the `channels` table.

Two reasons I can think of for creating a new table might be
1. It might be "neater" to store all blockchain-related data for a channel in a single table
2. It is possible to update the record corresponding to a channel `C` in the `finalization_status` table, while some other operation is updating the row corresponding to `C` in `channels`.

I'm not sure if either of these are a strong argument though?

Reply:
I think 1. was the main impetus behind this. It seemed like a good idea to keep the blockchain data separate similar to the `funding` table.

Reply:
should the table be called something as specific as `finalization_status` then?

Considering that it's easy to rename a table, I don't see a real issue with it just the same.

Reply:
I think this will probably get broadened to include all challenging info eventually, so it might make sense to rename it now.

Reply:
I've renamed this to `challenging_status` for now.